spec/solidus/gateway/braintree_gateway_spec.rb

```diff
@@ -88,6 +88,7 @@
           expect(capture).to be_success
           expect(capture.authorization).to be_present
           expect(capture.avs_result["code"]).to eq "M"
+          expect(capture.cvv_result["code"]).to eql("I")
```

Comment:
To calarify, cvv_result here is a hash returned by https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/billing/cvv_result.rb